build.sh

```diff
@@ -1,57 +1,89 @@
 #!/bin/bash
 set -ex
 
-yum update -y
-yum install -y \
+# Update this container
+echo "Yum updating container..." > /dev/null 2>&1
+yum -y update
+echo "Yum updating container...done" > /dev/null 2>&1
+
+# Set up env vars
+PYTHON_VER_YUM='36'
+PYTHON_VER='3.6'
+NUMPY_VER='1.13.3'
+SCIPY_VER='0.19.1'
+SKLEARN_VER='0.19.0'
+
+LAMBDA_PACKAGE_DIR='outputs/lambda-package'
+LIB_DIR="${LAMBDA_PACKAGE_DIR}/lib"
+LAMBDA_PACKAGE_ZIP='lambda-package.zip'
+LAMBDA_PACKAGE_ZIP_RELPATH="outputs/${LAMBDA_PACKAGE_ZIP}"
+
+SITE_PACKAGES_DIR="/usr/local/lib64/python${PYTHON_VER}/site-packages"
+
+echo "Yum installing non-pip packages..." > /dev/null 2>&1
+yum -y install \
     atlas-devel \
     atlas-sse3-devel \
     blas-devel \
+    findutils \
     gcc \
     gcc-c++ \
     lapack-devel \
-    python27-devel \
-    python27-virtualenv \
-    findutils \
+    python${PYTHON_VER_YUM}-devel \
     zip
+echo "Yum installing non-pip packages...done" > /dev/null 2>&1
+
+echo "Pip installing packages using local compilation for numpy and scipy..." > /dev/null 2>&1
+/usr/bin/pip-${PYTHON_VER} install --upgrade pip setuptools
```

Comment:
produced an error later in the script as pip 18.1 was installed but numpy and scipy installation looked for 9.0.3 ??

worked when this line was commented out